addons/website_forum/static/src/js/website_forum.share.js

```diff
@@ -4,8 +4,47 @@ import publicWidget from "@web/legacy/js/public/public_widget";
 import "@website/js/content/snippets.animation";
 import { renderToElement } from "@web/core/utils/render";
 
-// FIXME There is no reason to inherit from socialShare here
-var ForumShare = publicWidget.registry.socialShare.extend({
+const ForumShareBase = publicWidget.Widget.extend({
+    start() {
+        this.shareEl = this.$el[0].ownerDocument.querySelector(".o_share_prerendered");
+    }
+});
+
+publicWidget.registry.websiteForumHoverShare = ForumShareBase.extend({
+    selector: '.oe_social_share',
+    events: {
+        'mouseenter': '_onMouseEnter',
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _onMouseEnter() {
+        this.$el.popover({
+            content: this.shareEl,
+            placement: 'bottom',
+            container: this.$el,
+            html: true,
+            trigger: 'manual',
+            animation: false,
+        }).popover("show");
+
+        this.$el.off('mouseleave.socialShare').on('mouseleave.socialShare', function () {
+            var self = this;
+            setTimeout(function () {
+                if (!$(".popover:hover").length) {
+                    $(self).popover('dispose');
+                }
+            }, 200);
+        });
```

Comment:
Why is this needed?
I really don't like the usability of those popover (even before the pr), it's not a common UI thing, is it? It always glitches and closes while you are trying to move inside it etc.
Wouldn't it be better to have it open and close on click only?